x-pack/plugins/alerting/public/services/maintenance_windows_api/get.ts

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { HttpSetup } from '@kbn/core/public';
+import { AsApiContract, RewriteRequestCase } from '@kbn/actions-plugin/common';
+
+import { MaintenanceWindow } from '../../pages/maintenance_windows/types';
+import { INTERNAL_BASE_ALERTING_API_PATH } from '../../../common';
+
+const rewriteBodyRes: RewriteRequestCase<MaintenanceWindow> = ({ r_rule: rRule, ...rest }) => ({
+  ...rest,
+  rRule: { ...rRule },
```

Comment:
same question about just using `rRule,` instead of `rRule: { ...rRule },`

Reply:
Resolved in this commit https://github.com/elastic/kibana/pull/154669/commits/b2745dcd188cc86ffc5ecffcfc724d66cc13803f